include/netlink/route/link/macvlan.h

```diff
@@ -29,6 +29,9 @@ extern int		rtnl_link_macvlan_str2mode(const char *);
 extern char *		rtnl_link_macvlan_flags2str(int, char *, size_t);
 extern int		rtnl_link_macvlan_str2flags(const char *);
 
+extern char *		rtnl_link_macvlan_macmode2str(int, char *, size_t);
+extern int		rtnl_link_macvlan_str2macmode(const char *);
```

Comment:
symbols must be explicitly exposed in the ABI, that is: add them to libnl-route-3.sym.